### ISSUE TITLE
the library started failing (new update types)

### ISSUE THREAD

**Issue by ayrat555** [telegram-rs/telegram-bot#237]
Today the library started failing with :

```
Mar 09 10:28:22 vps695005 bash[29083]: [2021-03-09T09:28:22Z ERROR telegram_bot::api] error=no variant of enum UpdateKind found in flattened data at line 2 column 766
Mar 09 10:28:22 vps695005 bash[29083]: [2021-03-09T09:28:22Z ERROR telegram_bot::stream] request error error=no variant of enum UpdateKind found in flattened data at line 2 column 766
```
I think it's happening because telegram added two new update types (https://core.telegram.org/bots/api-changelog#march-9-2021)

**Comment by ayrat555** [comment]
in my case, the new events were causing failures. I added new enum variants like https://github.com/telegram-rs/telegram-bot/pull/221/files

**Comment by mann-ed** [comment]
TLDR; I added these updates to my fork as well. My fork has the fixes from Master. Not sure if it's worth a pull-request. https://github.com/telegram-rs/telegram-bot/compare/master...mann-ed:master

I created this my fork over a year ago to get inline response working, And i was just starting out learning RUST, so that is why there are several commits with same statement. I finally got it to work and went on to other things. Just recently my bot broke, and this is the fix.

**Comment by fxwiegand** [comment]
> TLDR; I added these updates to my fork as well. My fork has the fixes from Master. Not sure if it's worth a pull-request. [master...mann-ed:master](https://github.com/telegram-rs/telegram-bot/compare/master...mann-ed:master)
> 
> I created this my fork over a year ago to get inline response working, And i was just starting out learning RUST, so that is why there are several commits with same statement. I finally got it to work and went on to other things. Just recently my bot broke, and this is the fix.

Using `telegram-bot = { git = "https://github.com/ayrat555/telegram-bot", branch = "ayrat555/api-fixes-10" }` worked for me! Thanks a lot @mann-ed for the fix. It'd be great to have that merged somehow! 😄

**Comment by ayrat555** [comment]
I wrote a new telegram library for rust - https://github.com/ayrat555/frankenstein. It's easy to use and have a minimal number of dependencies. You can even use an HTTP client of your choice
